Sources/GRDBQuery/Queryable+DatabaseContext/QueryableOptions.swift

```diff
@@ -58,6 +58,12 @@ public struct QueryableOptions: OptionSet, Sendable {
     /// ``PresenceObservationQueryable`` types.
     public static let constantRegion = QueryableOptions(rawValue: 1 << 1)
     
+    /// By default, any error that occurs can be accessed using ``Query/Wrapper/error``
+    /// and is otherwise ignored.
+    /// With this option, assertNoFailure() is called on the combine publisher, which causes any error
+    /// to be treated as a fatalError which will terminate the app.
```

Comment:
Very good. Thanks for linking to the `error` property 👍

Let's just not talk about Combine:

```suggestion
    /// With this option, errors that happen while accessing the
    /// database terminate the app with a fatal error.
```

Reply:
Makes sense, thank you!